scripts/playermove.gd

```diff
@@ -147,6 +150,7 @@ func _physics_process(delta):
 			var impact_type = reported_body.get_class()
 			match impact_type:
 				"RigidBody": # physics object
+					emit_signal("hit_ball", reported_body.name, look_direction)
```

Comment:
Nice job adapting this functionality into multiplayer, online mp is new to me so it's awesome that it just takes one line of a signal to pass the necessary information like this.